foundry/external_data_architectures.py

```diff
@@ -25,3 +26,23 @@ def __getitem__(self, idx):
         
         return item
     
+class FoundryDataset_TF(Sequence):
+    def __init__(self, inputs, targets):
```

Comment:
Add in some arguments for these.

Reply:
not sure what you mean by this

Reply:
Whoops, that comment did make no sense. 

Add in some documentation for these arguments